Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -208,35 +246,70 @@ public void EvaluateActions()
                     [addresses[0]] = "A",
                     [addresses[1]] = "B",
                     [addresses[2]] = "C",
+                    [DumbAction.RandomRecordsAddress] = randomValue,
                 }.ToImmutableDictionary(),
                 dirty1
             );
 
-            Block<DumbAction> blockIdx2 = MineNext(
-                blockIdx1,
-                new[]
-                {
-                    Transaction<DumbAction>.Create(
-                        _fx.TxFixture.PrivateKey,
-                        new[] { MakeAction(addresses[0], 'D') }
-                    ),
-                    Transaction<DumbAction>.Create(
-                        _fx.TxFixture.PrivateKey,
-                        new[] { MakeAction(addresses[3], 'E') }
-                    ),
-                    Transaction<DumbAction>.Create(
-                        _fx.TxFixture.PrivateKey,
-                        new[]
-                        {
-                            new DumbAction(
-                                addresses[4],
-                                "RecordRehearsal",
-                                recordRehearsal: true
-                            ),
-                        }
+            Transaction<DumbAction>[] blockIdx2Txs =
+            {
+                Transaction<DumbAction>.Create(
+                    _fx.TxFixture.PrivateKey,
+                    new[] { MakeAction(addresses[0], 'D') }
+                ),
+                Transaction<DumbAction>.Create(
+                    _fx.TxFixture.PrivateKey,
+                    new[] { MakeAction(addresses[3], 'E') }
+                ),
+                Transaction<DumbAction>.Create(
+                    _fx.TxFixture.PrivateKey,
+                    new[]
+                    {
+                        new DumbAction(
+                            addresses[4],
+                            "RecordRehearsal",
+                            recordRehearsal: true
+                        ),
+                    }
+                ),
+            };
+            Block<DumbAction> blockIdx2 = MineNext(blockIdx1, blockIdx2Txs);
+            tuples = blockIdx2
+                .EvaluateActionsPerTx(dirty1.GetValueOrDefault)
+                .ToImmutableArray();
+            i = 0;
+            expectations = new[]
+            {
+                (0, 0, new[] { "A,D", "B", "C", null, null }),
+                (1, 0, new[] { "A,D", "B", "C", "E", null }),
+                (2, 0, new[] { "A,D", "B", "C", "E", "RecordRehearsal:False" }),
+            };
+            Assert.Equal(expectations.Length, tuples.Length);
+            foreach (var tuple in tuples)
+            {
+                var expect = expectations[i++];
```

Comment:
Rider says we can use deconstruction like:
```csharp
var (transaction, dumbAction, actionContext, accountStateDelta) = tuples[i];
var (item1, item2, strings) = expectations[i];
```
How about using deconstruction?
It looks more readable.

Reply:
Fixed by adding a distinct type: 19089e75db6287a8e9de6eb47b92f5d72eef0d1d.